src/overlays/actors/ovl_En_Hs/z_en_hs.c

```diff
@@ -102,67 +104,67 @@ void func_80952DFC(GlobalContext* globalCtx) {
     func_80151BB4(globalCtx, 0x10);
 }
 
-void func_80952E50(Vec3f* dst, Vec3f src, f32 arg2) {
-    Vec3f sp1C;
-    f32 temp_f0;
+void EnHs_UpdateChickPos(Vec3f* dst, Vec3f src, f32 playerOffset) {
```

Comment:
Not sure `playerOffset` is quite right. Doesn't quite make sense for the latter chicks as they offset relative to earlier chicks not to player. Personally I think just `offset` would be fine.